spring-cloud-gcp-samples/pom.xml

```diff
@@ -89,6 +90,22 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>release</id>
```

Comment:
Does it have to be in the `release` profile? We have a related thing for skipping deploy below, which is not in any profile. See:
```
			<plugin>
				<artifactId>maven-deploy-plugin</artifactId>
				<configuration>
					<skip>true</skip>
				</configuration>
			</plugin>
```

Reply:
It appears the Spring release used the standard Maven deploy plugin, where as Maven central uses the Nexus-staging plugin for some enhanced release capabilities. AFAICT the Nexus-staging one supercedes maven-deploy, but I'm not 100% on that.

As far as this being in the profile, I'm including it here because the root POM and the other non-inherited module (`spring-cloud-gcp-dependencies`)  also only includes the plugin in the `-P release` profile.